packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -593,7 +593,9 @@ class _CircularProgressIndicatorState extends State<CircularProgressIndicator> w
   }
 
   Widget _buildCupertinoIndicator(BuildContext context) {
-    return CupertinoActivityIndicator(key: widget.key);
+    final Color? tickColor = widget.backgroundColor ?? ProgressIndicatorTheme.of(context).circularTrackColor;
+    return tickColor == null ? CupertinoActivityIndicator(key: widget.key) :
```

Comment:
Couldn't this just be `return CupertinoActivityIndicator(key: widget.key, color: tickColor);`?

Reply:
Done